blocks/v2-explore-articles/v2-explore-articles.js

```diff
@@ -27,17 +49,32 @@ const CLASSES = {
   collageItemTitle: `${blockName}__collage-item-title`,
   showMoreButton: `${blockName}__show-more-button`,
   showMoreButtonWrapper: `${blockName}__show-more-container`,
+  noArticlesMessage: `${blockName}__no-articles-message`,
 };
 
 const docRange = document.createRange();
-const defaultAmount = 9;
-let currentAmount = 0;
+const defaultAmountOfArticles = 9;
+let totalArticleCount = 0;
+let offset = 0;
+let counter = 0;
+let appliedFilters = {};
```

Comment:
an object doesn't need to be a `let` unless you aren't going to mutate it
```suggestion
const appliedFilters = {};
```